nefertari/scripts/es.py

```diff
@@ -72,15 +74,18 @@ def main(argv=sys.argv):
     else:
         model_names = split_strip(options.models)
 
-    queue = get_processing_queue(model_names, processes, options)
-    pool = Pool(processes)
-    processors = map(lambda i: ESProcessor(options=options, queue=queue, pid=i),
+    BaseObject.metadata.bind.dispose()
```

Comment:
Yes, we need to close all existed connections for avoid issue with concurrent access to one conection. http://docs.sqlalchemy.org/en/latest/faq/connections.html#how-do-i-use-engines-connections-sessions-with-python-multiprocessing-or-os-fork